docs/modules/ROOT/pages/getting-started/installation.adoc

```diff
@@ -0,0 +1,64 @@
+= Installation
+
+This document explains how to install and configure KubeArchive in your Kubernetes cluster.
+
+== Prerequisites
+
+* A PostgreSQL instance
+* CertManager is installed on the Kubernetes cluster (+v1.9.1)
+* Knative Eventing is installed on the Kubernetes cluster (+v1.15.0)
+
+== Prepare the Database
+
+. Create a user and a password for KubeArchive to use.
+. Create a database for the user to use.
+. As the user, execute the `ddl.sql` we provide in the database, located at:
++
+[source,bash,subs="attributes"]
+----
+ifeval::["{page-component-display-version}" == "main"]
+https://github.com/kubearchive/kubearchive/releases/latest/download/ddl.sql
+endif::[]
+ifeval::["{page-component-display-version}" != "main"]
+https://github.com/kubearchive/kubearchive/releases/download/{page-component-display-version}/ddl.sql
+endif::[]
+----
++
+This file creates a PostgreSQL schema, a table for KubeArchive and other stuff KubeArchive
+requires.
+
+== Install and Configure KubeArchive
+
+. Apply the release YAML to install KubeArchive:
++
+[source,bash,subs="attributes"]
+----
+ifeval::["{page-component-display-version}" == "main"]
+kubectl apply -f https://github.com/kubearchive/kubearchive/releases/latest/download/kubearchive.yaml
+endif::[]
+ifeval::["{page-component-display-version}" != "main"]
+kubectl apply -f https://github.com/kubearchive/kubearchive/releases/download/{page-component-display-version}/kubearchive.yaml
+endif::[]
+----
+
+. Provide the credentials for the database in the `kubearchive-database-credentials` secret:
++
+[source,bash]
+----
+cat <<EOF > /tmp/patch.yaml
+stringData:
+  POSTGRES_URL: <your-postgresql-url>
+  POSTGRES_USER: <kubearchive-username>
+  POSTGRES_DB: <kubearchive-database>
+  POSTGRES_PASSWORD: '<kubearchive-password>'  # notsecret
```

Comment:
Is the `#notsecret` directive needed here?

Reply:
I will remove it and let's see, maybe the secrets scanner does not notice.